themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx

```diff
@@ -132,15 +126,15 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
                 <PageSection isFilled variant={PageSectionVariants.light}>
                     <Stack hasGutter>
                         <StackItem>
-                            <Title headingLevel="h2" size='2xl'>
+                            <Title headingLevel="h2" className="pf-u-mb-lg" size='xl'>
```

Comment:
If you wrap a <TextContent> around the <Title> then it should give you the correct vertical spacing without manually adding the utility class.

Reply:
I am not able to achieve the correct spacing with the <TextContent><Text> components as suggested. It doesn't appear to apply the correct spacing the spacing utility class provides. May need to meet and look over together.